cmd/loadTests.go

```diff
@@ -941,28 +974,40 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 			if err != nil {
 				logError(6, fmt.Sprintf("Unable to create ComponentDetectionQuery %s: %v", ComponentDetectionQueryName, err))
 				FailedCDQCreationsPerThread[threadIndex] += 1
+				MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedCDQCreationCounter)
 				increaseBar(cdqsBar, cdqsBarMutex)
 				continue
 			}
 			if cdq.Name != ComponentDetectionQueryName {
 				logError(7, fmt.Sprintf("Actual cdq name (%s) does not match expected (%s): %v", cdq.Name, ComponentDetectionQueryName, err))
 				FailedCDQCreationsPerThread[threadIndex] += 1
+				MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedCDQCreationCounter)
 				increaseBar(cdqsBar, cdqsBarMutex)
 				continue
 			}
 			if len(cdq.Status.ComponentDetected) > 1 {
 				logError(8, fmt.Sprintf("cdq (%s) detected more than 1 component", cdq.Name))
 				FailedCDQCreationsPerThread[threadIndex] += 1
+				MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedCDQCreationCounter)
 				increaseBar(cdqsBar, cdqsBarMutex)
 				continue
 			}
 
+
 			CDQCreationTimeSumPerThread[threadIndex] += cdqCreationTime
+			MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeGuage, metricsConstants.MetricCDQCreationTimeGauge, float64(cdqCreationTime))
 			if cdqCreationTime > CDQCreationTimeMaxPerThread[threadIndex] {
 				CDQCreationTimeMaxPerThread[threadIndex] = cdqCreationTime
 			}
 			SuccessfulCDQCreationsPerThread[threadIndex] += 1
+			MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeCounter, metricsConstants.MetricSuccessfulCDQCreationCounter)
 
+			for _, cdqInstance := range cdq.Status.Conditions {
+				if cdqInstance.Type == "Completed"{
+					TempTime := cdqInstance.LastTransitionTime.Sub(startTimeForCDQ)
+					MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeGuage, metricsConstants.MetricActualCDQCreationTimeGauge, float64(TempTime))
```

Comment:
```suggestion
					MetricsWrapper(MetricsController, metricsConstants.CollectorCDQ, metricsConstants.MetricTypeGuage, metricsConstants.MetricActualCDQCreationTimeGauge, TempTime.Seconds())
```
Instead of nanoseconds, please push seconds.